lib/almanac/cli.ex

```diff
@@ -0,0 +1,54 @@
+defmodule Almanac.CLI do
+  @moduledoc """
+  Handle the command line parsing and the dispatch to the various functions
+  that end up generating a table of the weather conditions from the specified
+  station ID provided as input by the user
+  """
+
+  def run(argv) do
+    argv
+    |> parse_args()
+    |> process()
+  end
+
+  @doc """
+  `argv` can be -h or --help, which returns :help.
+
+  Otherwise it is a station ID that will be used in the HTTP request to the
+  NOAA website.
+
+  Returns the station ID, or `:help` if help was given
+  """
+
+  def parse_args(argv) do
+    OptionParser.parse_head(argv,
```

Comment:
If `parse_head()` with its args fits on one line, or close to, that's the preferred formatting, esp. when sending to subsequent |> calls